HousingRepairsSchedulingApi.Tests/ServicesTests/Drs/DrsServiceTests.cs

```diff
@@ -242,7 +242,8 @@ public async void GivenCreateOrderResponse_WhenCreatingAnOrder_ThenBookingIdIsPr
             // Arrange
             this.soapMock.Setup(x => x.createOrderAsync(It.IsAny<createOrder>()))
                 .ReturnsAsync(new createOrderResponse(new xmbCreateOrderResponse
-                {
+                {
+                    status = responseStatus.success,
```

Comment:
See line 248 in DrsService - additional check has been added to make sure responseStatus is "success".